sandbox/testapp/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1.1 on 2018-09-03 13:13
+# Generated by Django 3.1 on 2023-07-28 10:00
```

Comment:
Is it necessary ? :) 
Django keeps compatibility of migration files with previous version.

Reply:
Ahah yes no problem about that :) 

I thought that Django supported migration files with previous Django version. Apparently it is not the case according to the [official documentation](https://docs.djangoproject.com/en/4.2/topics/migrations/#supporting-multiple-django-versions). Recreating migration files was the correct decision ^^